zrml/liquidity-mining/src/track_incentives_based_on_bought_shares.rs

```diff
@@ -69,7 +69,7 @@ where
             }
         }
 
-        <BlockBoughtShares<T>>::remove_all(None);
+        let _ = <BlockBoughtShares<T>>::clear(u32::max_value(), None);
```

Comment:
Multi-Block execution required here?

Reply:
should we do it in separate PR? This PR is only meant for upgrade polkadot version.